connect-button/src/main/res/values/dimens.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <dimen name="ifttt_text_padding_horizontal">72dp</dimen>
+    <dimen name="ifttt_text_padding_horizontal">@dimen/ifttt_connect_button_height</dimen>
```

Comment:
I would say we could leave this as `72dp` instead of setting it as the connect button height. Those two dimens are unrelated and if we decide to change height, this will change too

Reply:
They are related actually, the padding should match exactly how tall the connect button is.